server/utils/taskListUtils.test.ts

```diff
@@ -68,6 +72,19 @@ describe('taskListUtils', () => {
           })}" aria-describedby="eligibility-second-task" data-cy-task-name="second-task">Second Task</a>`,
         )
       })
+
+      it('should return a link of is-exceptional-case if risk tier not applicable', () => {
+        task.status = 'in_progress'
+        ;(isApplicableTier as jest.MockedFunction<typeof isApplicableTier>).mockReturnValue(false)
```

Comment:
it's clearer to use `when` like here: https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/78c23861471b7b60383d311daf6777907bc170f1/server/controllers/tasksController.test.ts#L67